src/index.js

```diff
@@ -1,6 +1,6 @@
 import express from 'express';
 import mongoose from 'mongoose';
-import routes from './routes';
+import routes from './routes/index';
```

Comment:
this file should be unmodified, originally this line was imported as,
```suggestion
import routes from './routes';
```